portal-web/docroot/html/common/themes/bottom_js.jspf

```diff
@@ -72,8 +72,10 @@
 	}
 </aui:script>
 
-<aui:script require="metal-dom/src/all/dom as dom" sandbox="<%= true %>">
-	var focusInPortletHandler = dom.delegate(
+<aui:script require="frontend-js-web/liferay/delegate/delegate.es as delegateModule" sandbox="<%= true %>">
```

Comment:
My understanding [based on the guidelines](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/aui_script.md) is that you don't need `sandbox` if you're using `use` or `require`, because both of them will effectively enclose the script contents inside a function and prevent leakage into the global namespace. So, you could just remove it. As proof, see this one, that shows one of the other files in this PR that you touched (this screenshot is from master, so it was `<aui:script require="metal-dom/src/dom as dom">`, but the principle is the same; the code winds up inside a function in a `Liferay.Loader.require` callback):

![Screenshot 2020-11-27 -112134-lmIMAVRP@2x](https://user-images.githubusercontent.com/7074/100438661-c4d41280-30a2-11eb-98e0-786a9a93858b.png)

Reply:
Now I know (after forgetting) what it's purpose is. Thanks alot @wincent so basically if we have either `use` or `require` we don't need `sandbox`.